eyecite/models.py

```diff
@@ -403,3 +403,24 @@ def compiled_regex(self):
         if not hasattr(self, "_compiled_regex"):
             self._compiled_regex = re.compile(self.regex, flags=self.flags)
         return self._compiled_regex
+
+
+@dataclass(frozen=True)
+class Resource:
+    """Thin resource class representing an object to which a citation can be
+    resolved."""
+
+    citation: FullCaseCitation
+
+    def __hash__(self):
```

Comment:
I think this can just be `return hash((self.citation.reporter, self.citation.volume, self.citation.page))`? There's an example at https://docs.python.org/3.5/reference/datamodel.html#object.__hash__

Reply:
Did this not pan out?

Reply:
My bad, I thought I made this change but I guess I didn't. Forthcoming.